runtime/runtime-core/common/src/aws/smithy/kotlin/runtime/io/SdkSink.kt

```diff
@@ -47,6 +47,5 @@ public interface SdkSink : Closeable {
      * Pushes all buffered bytes to their final destination and releases resources held by this sink.
      * It is an error to write to a closed sink. This is an idempotent operation.
      */
-    @Throws(IOException::class)
```

Comment:
**Question**: We still weren't able to figure out why it didn't like this? okio gets away with this in common and IIRC it is helpful since Intellij will mark methods with this as potentially blocking (which it is).